src/io/flutter/FlutterInitializer.java

```diff
@@ -113,6 +114,16 @@ public static void setVerboseLogging(boolean value) {
     properties.setValue(verboseLoggingKey, value);
   }
 
+  public static boolean isMemoryDashboard() {
```

Comment:
OK, sounds like a reasonable plan.

@B3rn475, I'll land this PR now; I may follow up with some tweaks to how we expose and toggle experimental features.